v0.15.0.2-osx-unsigned/akx20000/bitcoin-gold-osx-0.15-build.assert

```diff
@@ -0,0 +1,647 @@
+--- !omap
+- out_manifest: |
+    14041b7fd789c0ff03db6b84685831095da4efe4d8d42af8ba7cbe68682511a6  bitcoin-gold-0.15.0-osx-unsigned.dmg
+    0deb6536c9e8b6e4fde2066e4f18b114b8be3e1916a9e5abc9b85c3160ef9c12  bitcoin-gold-0.15.0-osx-unsigned.tar.gz
+    b3dcada9474499447a6c61c9c403c9f42f203a898bf0db8f634a6077cbc0f77e  bitcoin-gold-0.15.0-osx64.tar.gz
+    15ff83d38363c599befac76bf71ced3db9833559e2b7ef51730fa8ae09c4a458  src/bitcoin-gold-0.15.0.tar.gz
+- in_manifest: |-
+    f00c0c1e7ebab5775227444ea0f33f7f357f1b7bb323e22fcfe1177da4072c3b  bitcoin-gold-osx-0.15-desc.yml
+    b1207f2b5b5c05ea79b1c1d355163656b22a825e62d271b63bfff8827942d92c  MacOSX10.11.sdk.tar.gz
```

Comment:
Weird. The checksum of your sdk is not the same as mine and @wilsonmeier's, thought the outputs are same.

Reply:
That's wierd, I will do it again.